.github/workflows/pytest.yml

```
@@ -77,6 +77,11 @@ jobs:
           pytest -n 2 --dist loadfile --nbval --current-env --cov-append tutorials/
           echo "Exited with '$?'"
 
+      - name: test with WeldxFile
+        if: (matrix.os == 'ubuntu-latest') && (matrix.py == '3.8')
+        run: |
+          pytest -n2 --dist loadfile --weldx-file-rw-buffer --weldx-file-rw-buffer-disp-header
```

```suggestion
          pytest ./weldx/tests/asdf_tests -n2 --dist loadfile --weldx-file-rw-buffer --weldx-file-rw-buffer-disp-header
```

I think it is enough to run in this subfolder since that is basically where everything asdf related happens
preivious coverage was 89% in this run let's see
https://github.com/BAMWelDX/weldx/pull/469/checks?check_run_id=3375408820#step:11:68

---

Good hint! Thank you